src/lib/processor.js

```diff
@@ -12,12 +12,19 @@ module.exports = {
          options.concurrency || 1,
          self._getRunner(options),
          function runTask(error, results) {
-           if (error) {
-             logger.debug('error: ', error);
+           let err = error || results.map(el => el.error).filter(el => el);
```

Comment:
Like most async methods, `async.mapLimit` stops when an error occurs. 
Converting the results to the signature `(null, {error, results})` fixes the problem of the process chain breaking.
I still left the old signature `(error, results)` intact so not to break compatibility.